src/audio/lib/queue/QueueStore.ts

```diff
@@ -47,7 +48,7 @@ export class QueueStore extends Collection<string, Queue> {
 		for (const command of commands) {
 			this.redis.defineCommand(command.name, {
 				numberOfKeys: command.keys,
-				lua: readFileSync(resolve(__dirname, 'scripts', `${command.name}.lua`)).toString()
+				lua: readFileSync(resolve(rootFolder, 'scripts', 'audio', `${command.name}.lua`)).toString()
```

Comment:
Remove `"postbuild": "node scripts/copy-lua-files.mjs",` from `package.json`, as well as the script then.